src/Engine/ProtoCore/Reflection/GraphicDataProvider.cs

```diff
@@ -141,8 +141,17 @@ internal List<IGraphicItem> GetGraphicItems(DSASM.StackValue svData, RuntimeCore
             return null;
         }
 
+        //Store marshaller for repeated calls to GetCLRObject.  Used for short-circuit of re-allocation of marshaller / interpreter object.
+        private static Interpreter interpreter;
+        private static ProtoFFI.FFIObjectMarshaler marshaler;
```

Comment:
Why do these need to be static?

Reply:
They are static because you only need one Interpreter and FFIObjectMarshaller for the `GraphicDataProvider` class until the runtime is reset.  If you made the instances you would allocate permanently an interpreter and marshaller for every mirror data object and that would be worse than temporary allocation

Reply:
I see that the `graphicdataprovider `member for `mirrordata` is static too so it looks like there will only be one instance of `graphicdataprovider ` ever but I guess it doesn't hurt to make these static here as well.